providers/flagd/README.md

```diff
@@ -42,19 +42,37 @@ openfeature.SetProvider(provider)
 
 In the above example, in-process handlers attempt to connect to a sync service on address `localhost:8013` to obtain [flag definitions](https://github.com/open-feature/schemas/blob/main/json/flagd-definitions.json).
 
+#### Offline mode
+
+In-process resolvers can also work in an offline mode.
+To enable this mode, you should provide a valid flag configuration file with the option `WithOfflineFilePath`.
+
+```go
+provider := flagd.NewProvider(
+        flagd.WithInProcessResolver(),
+        flagd.WithOfflineFilePath(OFFLINE_FLAG_PATH))
+openfeature.SetProvider(provider)
+```
+
+Provider will attempt to detect file changes but this is a best effort attempt as file system events are different from os to os.
+This mode is useful for local development, tests and offline applications.
+For a full-featured, production-ready file-based implementation, use the RPC evaluator in combination with the flagd standalone application, which can be configured to watch files for changes.
+
```

Comment:
maybe we can add a disclaimer that this mode wins over the grpc one if both are configured 🤔

Reply:
Good suggestion. This is done - see last section of the offline mode - https://github.com/open-feature/go-sdk-contrib/blob/71322ed8fdb4d29c23b46f81313c0223e7deeba1/providers/flagd/README.md#offline-mode